index.html

```diff
@@ -102,9 +103,67 @@ <h3 class="project__name">Tonic</h3>
           <li class="languages"><a href="#">css </a></li>
           <li class="languages"><a href="#">javaScript </a></li>
         </ul>
-        <a href="#" class="project__link"> See Project </a>
+        <a href="javascript:void(0)"  id="project_1" class="project__link" onclick="open_popup(this.id)"> See Project</a>
       </div>
     </section>
+    <!-- Pop up section Implementation Started -->
+    <div id="overlay-popup">
+      <div id="overlay-content-popup">
+        <section class="work" id="portfolio_popup">
+          <div class="popup-header">
+            <h3 class="project__name" id="popup_title">Tonic</h3>
+            <a href="javascript:void(0)" onclick="close_popup()">&times;</a>
+          </div>
+            
```

Comment:
- [ ] Modal window is looking great! However, we need to address some requirements that are missing:

![Captura de pantalla 2022-05-11 a la(s) 8 21 04 p  m](https://user-images.githubusercontent.com/83717285/167973291-53f42762-d31d-4122-aab7-aa45b2d2f7ab.png)

Right now your `Work` section and pop-up window are hardcoded, this means that they are still in the HTML file, the goal here is to create the `Work` section and pop-up window using JavaScript using a `projects` array, in order to achieve this there's should be no projects information in the HTML file.

What you are doing with the pop-up window is to hide/show it, but it should be created using Javascript.

![Captura de pantalla 2022-05-11 a la(s) 8 21 58 p  m](https://user-images.githubusercontent.com/83717285/167973996-1dbb2a09-3c83-443d-888d-602af3a232f2.png)

I would recommend creating a function that gets the data of the project from the `projects` array and creates the HTML elements using DOM manipulation with JavaScript.

Reply:
They are not hardcoded.  They are coming from JavaScript array. The text you are showing me are the default values.
![image](https://user-images.githubusercontent.com/13853450/167982775-07f7806f-ffa7-45c6-8b28-91670cf3b8e1.png)
just change any value in the JS object array and you will see its effects.
Let me Explain: When some one click on the button, The id of the button does to the get_project(val) function and it gets its content from that place. 
![image](https://user-images.githubusercontent.com/13853450/167983009-6b61976e-4a1c-4c5d-9e67-1d67ada99b62.png)